opennlp-tools/src/main/java/opennlp/tools/util/FileUtil.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package opennlp.tools.util;
+
+import java.io.File;
+
+public class FileUtil {
+
+  /**
+   * delete the specified directory and its child directories and files
+   * @param file specify the file or directory to be deleted
+   */
+  public static void deleteDirectory(File file) {
```

Comment:
Since this is only used by tests should this class also go under the test package?

Reply:
+1 to not have it as part of public API, should be in test.